README.md

```diff
@@ -46,12 +46,274 @@ setting up a database inside an enclave is very well possible. This sample shows
 table, insert records into it, and select records from the table. This also shows how persisted records can be retrieved by
 the enclave once the host is re-started.
 
-### Initial SetUp
+# Write your first Conclave application in 5 mins
 
-In order to compile, and run these samples successfully, kindly download the [conclave SDK](https://www.conclave.net/get-conclave/) and update 
-the conclaveRepo property in gradle.properties to point to this path.
+This tutorial can be used to get started with Conclave in 5 mins. We will create a sample in which the client can send
+a string to the enclave, the enclave will reverse it and send it back to the client.
```

Comment:
"... and the machine on which the enclave is running will not be able to discover what the string was, even though the reversal logic is executing on that machine!" (assuming we show them how to do release mode?)

Reply:
done